Jenkinsfile-Native

```diff
@@ -94,15 +94,15 @@ def createBuildStage(name, definition) {
                     sh """
                       set -eu 
                       env | sort
-                     ./Configure ${definition.sslPlatform} --prefix=/usr --openssldir=/usr/lib/ssl no-ssl    no-tests no-ui-console no-unit-test 
+                     ./Configure ${definition.sslPlatform} --prefix=/usr --libdir=lib --openssldir=/usr/lib/ssl no-ssl no-tests no-ui-console no-unit-test
```

Comment:
What's the need for this change?

Reply:
Debian builds for openssl 3.1.1 were configured with multilib postfix, so libs were being written to `/lib64` instead of `/lib`. The recommendation I found was to use libdir (here: https://github.com/openssl/openssl/blob/master/INSTALL.md#libdir)

Reply:
Here's the description at that link too:

> The name of the directory under the top of the installation directory tree (see the --prefix option) where libraries will be installed. By default this is lib. Note that on Windows only static libraries (*.lib) will be stored in this location. Shared libraries (*.dll) will always be installed to the bin directory.
> 
> Some build targets have a multilib postfix set in the build configuration. For these targets the default libdir is lib<multilib-postfix>. Please use --libdir=lib to override the libdir if adding the postfix is undesirable.
>